tox.ini

```diff
@@ -9,24 +9,29 @@
 # something sensible.
 
 [tox]
-envlist = py{27,3}
+envlist = py{3}
 
-[testenv:py27]
-commands =
-    coverage run --source=./nmosregistration -m unittest discover -s tests
-    coverage annotate
-    coverage report
-deps =
-    coverage
-    mock
-    testcontainers
+; [testenv:py27]
+; commands =
+;     coverage run --source=./nmosregistration -m unittest discover -s tests
+;     coverage annotate
+;     coverage report
+; deps =
+;     coverage
+;     mock
+;     testcontainers
 
 [testenv:py3]
 commands =
-    coverage run --source=./nmosregistration -m unittest discover -s tests
+    ; coverage run --source=./nmosregistration -m unittest discover -s tests
+    coverage run --source=./nmosregistration -m pytest
```

Comment:
Commit should have referenced move to pytest for more reliable (blocking/non-concurrent) integration testing.